### ISSUE TITLE
Use bracketed ${vars} inline in double-quotes.

### ISSUE THREAD

**Issue by Aquaricat** [UnicornHeartClub/Issues#90]
Now that the system has been adjusted to allow `${brackets}` in variable names, it opens the possibility for a more convenient string generation for `!say`.  Instead of 
```
!say "String fragment " ${var} " another fragment " ${var2} " closing fragment."
```
could we take a page from bash parsing and just say
```
!say "String fragment ${var} another fragment ${var2} closing fragment."
```
thus reducing the double-quote tax and streamlining the line, while also making it easier to read and less likely that an opening or closing quote will be missed?

**Comment by Aquaricat** [comment]
Did some testing with `"${vars}"`, they're working good.

I also tried a couple simple `@{me}.attributes`, but they failed silently with no output and no error message.  Not sure if this is implemented yet, though, just know you were working on them.

**Comment by tlackemann** [comment]
What particularly is failing? (console has the input, program output, and execution output) I'm trying token attributes now and they appear to be working but I may not be trying what you're trying :)

**Comment by Aquaricat** [comment]
This is actually working as of the String Interpolation fixes of 2018.1.13.  Previously it gave no errors or output of any kind in UI, and I didn't think to check the console.  D'oh.  Closing!
